recognition/scoring.py

```diff
@@ -202,12 +202,21 @@ def calc_wer(self):
 
 
 class Hub5ScoreJob(Job):
-    def __init__(self, ref, glm, hyp):
+    __sis_hash_exclude__ = {"sctk_binary_path": None}
+
+    def __init__(
+        self,
+        ref: Union[tk.Path, str],
+        glm: Union[tk.Path, str],
+        hyp: Union[tk.Path, str],
```

Comment:
```suggestion
        ref: tk.Path,
        glm: tk.Path,
        hyp: tk.Path,
```
We want to disallow using strings for inputs that are a file/folder in the file system.

Also `self.sctk_binary_path.get_path()` will not work on `str`.
Request changes w.r.t. this comment.